### PR TITLE
EDS Drawer

### PR DESCRIPTION
**Pull request by roblevintennis**
**Description:**
- [Asana Task]()

- [x] Escape functionality
- [x] Drawer container (to allow ESC to work). See the Modal container in Brad's modal stuff
- [x] Cancel buttton should toggle

### CODE REVIEW

File: src/nora/components/NoraDrawer/NoraDrawer.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  return (
+    <Drawer
+      className={styles.NoraDrawer}
+      onDismiss={onDismiss}
+      isOpen={isOpen}
+      position={position}
+    >
+      <header className={styles.Header}>
+        <button onClick={() => onDismiss(false)} className={styles.Close}>
+          {closeCopy}
```
Comment (stephaniel96) [inline src/nora/components/NoraDrawer/NoraDrawer.js]: Not sure if it's the styling but it seems like the focus on the button or the text might not be centered properly?
image

File: src/nora/components/NoraDrawer/NoraDrawerExample.module.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+section {
```
[the remark this answers is not in the document]
Comment (roblevintennis) [reply src/nora/components/NoraDrawer/NoraDrawerExample.module.scss]: Oh yeah, that must have some how worked via the name of the element. I'll change it to `.Section`

File: src/nora/components/NoraDrawer/NoraDrawer.js (hunk 1 above)
Comment (roblevintennis) [reply src/nora/components/NoraDrawer/NoraDrawer.js]: image

in latest

Comment (stephaniel96) [reply src/nora/components/NoraDrawer/NoraDrawer.js]: woot woot!

**Pull request merged**